.circleci/config.yml

```diff
@@ -335,23 +339,45 @@ jobs:
           paths:
             - build
 
-  dist_bintray_nightly:
-    machine: true
+  dist_obs_nightly:
+    docker:
+      - image: straightshoota/osc
     steps:
       - attach_workspace:
           at: /tmp/workspace
       - run:
-          no_output_timeout: 20m
           command: |
             cd /tmp/workspace/distribution-scripts
             source build.env
-            cd bintray
-            ./publish-nightly.sh $CRYSTAL_VERSION $(date '+%Y-%m-%d') \
+            # osc may try to use a pager, even if stdout is not a TTY,
+            # cat avoids stalling on that.
+            export PAGER=cat
```

Comment:
I see the `straightshoota/osc` docker image already has a `PAGER=cat`. Is this needed?

Reply:
Ah right, we don't need both.